utils/_context/_scenarios/docker_ssi.py

```diff
@@ -141,8 +141,8 @@ def fill_context(self, json_tested_components):
             logger.stdout(f"{key}: {self._tested_components[key]}")
 
     def post_setup(self):
-        logger.stdout("--- Waiting for all traces to be sent to test agent ---")
-        time.sleep(5)  # wait for the traces to be sent to the test agent
+        logger.stdout("--- Waiting for all traces and telemetry to be sent to test agent ---")
+        time.sleep(60)  # wait for the traces and telemetry to be sent to the test agent
```

Comment:
I think waiting for 60 is too much. In most cases, this is not necessary
Could you try to make it configurable for this case?

Reply:
We have to wait long enough for telemetry data to flush, which could be up to 60 seconds :/

If we are able to shut down the app, we can probably force a flush.